docs/references/http_api_reference.md

```diff
@@ -764,7 +764,7 @@ curl --request GET \
 
 ## Show variables
 
-Gets variables.
+Get variables.
```

Comment:
Needs further clairification.

Reply:
Yes, variables in system.

Reply:
These variables indicate the status of infinity. And one more document are needed to clarify these variables.